pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -117,7 +117,7 @@ func (m *defaultVirtualCenterManager) RegisterVirtualCenter(ctx context.Context,
 	}
 
 	// Note that the Client isn't initialized here.
-	vc := &VirtualCenter{Config: config}
+	vc := &VirtualCenter{Config: config, ClientMutex: &sync.Mutex{}}
```

Comment:
do we need similar change at https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/74b42e954bf5d92940e21bf7bddeb9f38e8025b3/pkg/common/cns-lib/volume/listview_test.go#L126 ??

Reply:
This is just a test and we do not need the mutex here like we do not have Config set here.

Reply:
Added ClientMutex for consistency.